android/app/src/main/assets/config.json

```diff
@@ -1,5 +1,5 @@
 {
   "Email": "dev@fossasia.org",
   "App_Name": "Open Event",
-  "Api_Link": "https://eventyay.com/api/v1/events/6/"
+  "Api_Link": "https://api.eventyay.com/v1/events/69/"
```

Comment:
This is still a test server. The database may get flushed anytime. Don't depend on a specific event please.